pwiz_tools/Skyline/Skyline.resx

```diff
@@ -3915,4 +3915,16 @@
   <data name="&gt;&gt;$this.Type" xml:space="preserve">
     <value>pwiz.Skyline.Util.FormEx, Skyline, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null</value>
   </data>
+  <data name="synchMzScaleToolStripMenuItem.Size" type="System.Drawing.Size, System.Drawing">
+    <value>193, 22</value>
+  </data>
+  <data name="synchMzScaleToolStripMenuItem.Text" xml:space="preserve">
+    <value>Sync m/z scale</value>

```

Comment:
We use the full English term "Synchronize" for "Synchronize Zooming" on the chromatogram graphs. Stay consistent. Also it should be "Synchronize m/z Scale". You can't capitalize "m/z" (when possible it should be italicized), but otherwise menu items are Title Case.

Reply:
Corrected.